Terran-One/cw-simulate-ui
# Clicking on Codes now takes you to Codes page.

## Issue link

[WL-432](https://terran-one.atlassian.net/browse/WL-432)

## Description

Click on codes menu item item now takes you to codes and instances tab.

## Test steps

1. Add Chain
2. Now in that chain upload a contract.
3. Now click on Codes, any other tab and then again Codes on menu item. It should take you to Codes and Instances page every time you click Codes menu item.
